source/js/plugins/Pixel.js/source/export.js

```diff
@@ -16,6 +19,44 @@ export class Export
         this.uiManager = uiManager;
     }
 
+    createBackgroundLayer() {
+        //If export button has already been clicked, remove that background layer from layers
+        if (this.layers.length === 4) { 
+            this.layers.pop();
+        }
+        let backgroundLayer = new Layer(4, new Colour(242, 0, 242, 1), "Background Layer", this.pixelInstance, 0.5, this.pixelInstance.actions);
```

Comment:
Avoid lines longer than 80 characters. When a statement will not fit on a single line, it may be necessary to break it. Place the break after an operator, ideally after a comma. A break after an operator decreases the likelihood that a copy-paste error will be masked by semicolon insertion. The next line should be indented 8 spaces.

Reply:
Got it, thanks!